CMakeLists.txt

```diff
@@ -212,7 +212,17 @@ find_package(Threads)
 if(NOT CMAKE_SYSTEM_NAME STREQUAL "WASI")
     list(APPEND qjs_libs ${CMAKE_THREAD_LIBS_INIT})
 endif()
-if(NOT MSVC)
+
+# try to find libm
+string(TOLOWER "${CMAKE_C_COMPILER}" CC_LOWER)
+string(FIND "${CC_LOWER}" "tcc" TCC_SUBSTRING)
```

Comment:
This won't work. You can check for TCC like so: `CMAKE_C_COMPILER_ID STREQUAL "TinyCC"`